Scala/Scala.sublime-syntax

```diff
@@ -92,6 +92,18 @@ contexts:
     - include: late-operators
 
   block-comments:
+    - match: (^\s*)?/\*\*
+      scope: punctuation.definition.comment.scala
+      push:
+        - meta_scope: comment.block.documentation.scala
+        - match: \*/\s*$
```

Comment:
Won't this potentially run into an error like what was fixed with #596? Perhaps if you want to enforce nothing on the same line, mark everything other than a space after `*/` as invalid?